openrest4j-api/src/main/java/com/openrest/v1_1/Restaurant.java

```diff
@@ -48,7 +48,7 @@ public Restaurant(String id, String alias, String affiliateId, Map<String, Strin
                       Map<String, String> media, List<AppInfo> apps, Map<String, String> properties,
                       Boolean closed, Set<Product> products, Map<String, Double> features, String currentLocationId,
                       String defaultLocationIdAtTimeOfMigration, LocalizedString locationName, List<Location> locations, String defaultDispatchType, String posProviderId,
-                      String initialBranchId) {
+                      String zeroLocationsBranchId) {

```

Comment:
can you change it to zeroLocationsRestaurantLocationsId
And also in Location -> restaurantLocationsId instead of branchId

Reply:
restaurantLocation**s**Id should be restaurantLocationId